sdk/nodejs/output.ts

```diff
@@ -460,6 +472,66 @@ function getResourcesAndDetails<T>(allOutputs: Output<Unwrap<T>>[]): [Resource[]
     return [allResources, isKnown, isSecret];
 }
 
+/**
+ * Unknown represents a value that is unknown.
```

Comment:
Consider linking to the value as defined in the engine.   Consider additional docs.  i.e. thins like "normally users will not see these values in outputs.  specifically, because an output that contains this will be thought of as `isKnown=false`.  However, we do flow these along through outputs with simple property accesses like yadda yadda yadda you get the idea".

Reply:
Done.